### PR TITLE
chore(docs): update bartholomew module

### PR DESCRIPTION
**Pull request by vdice**
Updates the `bartholomew.wasm` module for the docs website for compatibility with spin at HEAD.  In particular, uses a more recent version of bartholomew that has dropped wasi_experimental_http, as Spin has.

Fixes the behavior seen currently when trying to run the docs:
```
$ spin up
Error: unknown import: `wasi_experimental_http::close` has not been defined
Error: exit status: 1
```

The bartholomew module was built from the [most recent commit](https://github.com/fermyon/bartholomew/commit/485629b3f3e5447d01ca3a43970ec127df3f236c) of fermyon/bartholomew (as of writing).

### CODE REVIEW

**Review by itowlson — APPROVED** [review]
Worked for me with current Spin HEAD

**Comment by tpmccallum** [comment]
Hi @vdice 
Just tested this also, works on Spin tip.

**Spin**

```bash
# Install Spin tip
cd ~
git clone https://github.com/fermyon/spin.git
cd spin
cargo install --locked --path .
```

**Bartholomew**

```bash
# Use the bartholomew-site-template (clone personal copy of template)
git clone https://github.com/tpmccallum/test_bart_wasm.git
# Correctly received the error
tpmccallum@192-168-1-19 test_bart_wasm % spin up --follow-all
Error: unknown import: `wasi_experimental_http::close` has not been defined
Error: exit status: 1
```

**Updated Bartholomew Wasm**

```bash
# Fetch and use new bartholomew.wasm file from PR https://github.com/fermyon/spin/pull/731/files
cp -rp ~/Downloads/bartholomew.wasm ~/test_bart_wasm/modules/
spin up --follow-all  
```

Screen Shot 2022-09-02 at 9 54 18 am

Screen Shot 2022-09-02 at 9 54 01 am

TL;DR LGTM

**Review by tpmccallum — APPROVED** [review]

**Review by radu-matei — APPROVED** [review]

**Comment by vdice** [comment]
Thank you for testing @tpmccallum!
